packages/xod-client-electron/src/arduinoDependencies/messages.js

```diff
@@ -38,4 +38,10 @@ export default {
     solution: 'Now you are able to upload the program',
     persistent: true,
   }),
+  ARDUINO_CLI_NOT_FOUND: ({ path }) => ({
+    title: 'Arduino-cli not found',
+    note: `Tried to find arduino-cli by the path: ${path}`,
```

Comment:
:scroll: ~by~ on

As far as I see, there are chances the message will be quite meaningless: “Tried to find arduino-cli by the path: arduino-cli”